src/clr-core/styles/list/_list.scss

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+$list-indent: $cds-token-layout-space-md;
+
+@mixin renderListStyles($type: ul, $styles: null) {
+    $ul-default: disc;
+    $ol-default: decimal;
+
+    @if $styles != null {
+        @each $style in $styles {
+            @if $style == $ul-default or $style == $ol-default {
+                #{$type}[cds-list],
+                #{$type}[cds-list='#{$style}'] {
+                    list-style-type: $style;
+                }
+            } @else {
+                #{$type}[cds-list='#{$style}'] {
+                    list-style-type: $style;
+                }
+            }
+        }
+    }
+}
+
+%kill-list-styles {
+    padding-left: 0 !important;
+    margin-left: 0 !important;
+    // zero-width blank space added to help VoiceOver screen reader read a list-style-type none list
+    list-style: '\200B' !important;
+    list-style-position: inside !important;
+}
+
+ul[cds-list]:not([cds-list='unstyled']),
+ol[cds-list] {
+    list-style-position: outside;
+    padding-left: $list-indent;
+}
+
+@include renderListStyles(ul, disc circle square);
+
+@include renderListStyles(ol, arabic-indic armenian bengali cjk-earthly-branch cjk-heavenly-stem cjk-ideographic decimal decimal-leading-zero devanagari ethiopic-halehame-am ethiopic-halehame-ti-er ethiopic-halehame-ti-et georgian gujarati gurmukhi hangul hangul-consonant hiragana hiragana-iroha kannada katakana katakana-iroha khmer korean-hangul-formal korean-hanja-formal korean-hanja-informal lao lower-alpha lower-greek lower-latin lower-roman malayalam mongolian myanmar oriya persian simp-chinese-formal simp-chinese-informal telugu thai upper-alpha upper-latin upper-roman urdu);
+
+ul[cds-list='unstyled'] {
```

Comment:
If the majority of ordered list styles is based on regions couldn't this be a root variable that the app sets based on the region they are serving? If its on root it should work with the IE polyfill and then we don't have to send region specific styles to every client.

```scss
// _list.scss
ol[cds-list] {
  list-style-type: var(--cds-ordered-list-style-type, decimal);
}
```

```scss
// app css dev can override 
:root {
  --cds-ordered-list-style-type: thai;
}
```

Or if that doesnt work seems like region specific styles could be overridden globally for the entire application similar to how we handle i18n globally.

```scss
// app dev adds this css based on their supported region
ol[cds-list] {
  list-style-type: thai;
}
```